AutoGVP/01-annotate_variants_CAVATICA_input.R

```diff
@@ -56,7 +56,7 @@ option_list <- list(
               help = "variant_summary file (format: variant_summary_2023-02.txt)"),
   make_option(c("--submission_summary"), type = "character",
               help = "specific submission summary file (format: submission_summary.txt)"),
-  make_option(c("--gnomad_variable"), type = "character",default = "Freq_gnomAD_genome_ALL",
+  make_option(c("--gnomad_variable"), type = "character",default = "gnomad_3_1_1_AF_non_cancer",
```

Comment:
I don't think all files will have this default column - @jungkim2 can you provide all of the default column annots with VEP (ie. - is gnomad any version always run, is 1000g always run) - for this column, i think @migbro specifically named it this way, so it will not pertain to user input

Reply:
probably also better to name this `--pop_freq_variable`

Reply:
For VEP annotation, I don't necessarily annotate with gnomAd and/or 1000g and --gnomad_variable filtering is not required. So it would be up to each user to be annotated right?

Reply:
I agree with the naming --pop_freq_variable as it could be gnomad/1000g/ESP/etc

Reply:
ok gotcha - yes, we were seeing run times on smaller files of 3 hours on ec2 with no filtering, but maybe we need to customize our EC2 template, and for this @naqvia we should leave this as optional - no default

Reply:
I think in order to make this usable and universal. We are observing that without filtering, we are running into long run times and memory-related crashes (cc @rjcorb). So, I think if we do not require a filter, we need at least to add a disclaimer or a warning for the end-user as the tool will likely crash and/or take a long time to run.